modules/administration/pages/tshoot-sync.adoc

```diff
@@ -32,3 +43,52 @@ Please run spacewalk-repo-sync in interactive mode to import it.
 
 To resolve the problem, you need to import the GPG key to {productname}.
 For more on importing GPG keys, see xref:administration:repo-metadata.adoc[].
+
+
+Checksum Mismatch::
+If a checksum has failed, you might see an error like this in the [path]``/var/log/rhn/reposync/*.log`` log file:
++
+----
+Repo Sync Errors: (50, u'checksums did not match
+326a904c2fbd7a0e20033c87fc84ebba6b24d937 vs
+afd8c60d7908b2b0e2d95ad0b333920aea9892eb', 'Invalid information uploaded
+to the server')
+The package microcode_ctl-1.17-102.57.62.1.x86_64 which is referenced by
+patch microcode_ctl-8413 was not found in the database. This patch has
+been skipped.
+----
++
+You can resolve this error by running the synchronization from the command prompt with the [command]``-Y`` option:
++
+----
+spacewalk-repo-sync --channel <channelname> -Y
+----
++
+This option verifies the repository data before the synchronization, rather than relying on locally cached checksums.
+
+
+Timeout::
+If you are synchronizing a {redhat} client with a very busy server, you might see a timeout error like this:
++
+----
+Errno 256 - No more mirrors to try
```

Comment:
This is not the timeout error. This is general message yum print when something goes wrong.
Currently i am not sure how to really detect a timeout error.

Reply:
Removed the error line.